app/views/gitter/chat_button.erb

```diff
@@ -0,0 +1,17 @@
+<% if control.language.downcase == 'ruby' %>
```

Comment:
Hope to not need to use capitalize or downcase eventually at all.  The track control data should be the formal name of the language.